lib/testium-core.js

```diff
@@ -73,7 +85,31 @@ function initTestium() {
     };
   }
 
-  return launchAll(config).then(createFromProcesses);
+  function verifySelenium(procs) {
+    return new Bluebird(function(resolve, reject) {
+      var seleniumUrl = config.get('selenium.serverUrl');
+      debug('Verify selenium: ', seleniumUrl);
+      var req = http.get(seleniumUrl + '/status', function(res) {
+        debug('Selenium /status: ', res.statusCode);
+        resolve(procs);
+      });
+      req.on('error', function(error) {
+        var oldStack = error.stack;
+        oldStack = oldStack.substr(oldStack.indexOf('\n') + 1);
```

Comment:
Honest answer: This was blindly copied from testium. Created https://github.com/testiumjs/testium-core/issues/9 to look into it.